pygmt/base_plotting.py

```diff
@@ -1918,3 +1918,197 @@ def update_pointers(data_pointers):
             with file_context as fname:
                 arg_str = " ".join([fname, build_arg_string(kwargs)])
                 lib.call_module("meca", arg_str)
+
+    @fmt_docstring
+    @use_alias(
+        A="vector",
+        B="frame",
+        C="cmap",
+        D="rescale",
+        E="uncertainty_color",
+        G="facecolor",
+        J="projection",
+        L="line",
+        N="no_clip",
+        R="region",
+        S="scaling",
+        U="timestamp",
+        V="verbose",
+        W="pen",
+        X="xshift",
+        Y="yshift",
+    )
+    @kwargs_to_strings(R="sequence", i="sequence_comma")
+    def velo(self, data=None, scaling=None, **kwargs):
+        """
+        Plot velocity vectors, crosses, and wedges
+
+        Reads data values from files, numpy array or panda dataframe and will plot velocity
+        arrows on a map. Most options are the same as for plot, except -S.
+
+        Must provide  *data* and *scaling*.
+
+
+        Full option list at :gmt-docs:`supplements/geodesy/velo.html`
+
+        {aliases}
+
+        Parameters
+        ----------
+        data : str or 2d array or dataframe
+            Either a data file name, a 2d numpy array or a panda dataframe.
+
+        {J}
+
+        {R}
+
+        scaling: str
+            Selects the meaning of the columns in the data file and the figure
+            to be plotted. In all cases, the scales are in data units per length
+            unit and sizes are in length units (default length unit is
+            controlled by `PROJ_LENGTH_UNIT <https://docs.generic-mapping-tools.org/latest/gmt.conf.html#term-proj-length-unit>`_
+            unless **c**, **i** , or **p** is appended).
```

Comment:
```suggestion
            to be plotted. In all cases, the scales are in data units per
            length unit and sizes are in length units (default length unit is
            controlled by :gmt-term:`PROJ_LENGTH_UNIT` unless **c**, **i** , or
            **p** is appended).
```

Recently added a `:gmt-term:` [sphinx-extlink](https://www.sphinx-doc.org/en/master/usage/extensions/extlinks.html) at #676 so we can take away the long URL! Also helped you to trim the lengths to 79 characters here.

Reply:
Great I didnt realised that!